README.md

```diff
@@ -1,3 +1,8 @@
+# Deprecation Warning
+
+**This repository is no longer maintained. Please use the [official envoy repository](https://github.com/envoyproxy/envoy)**
```

Comment:
Please use the [upstream envoy repository](https://github.com/envoyproxy/envoy) which now has support for the Windows platform.